package-lock.json

```diff
@@ -1,8 +1,9889 @@
 {
   "name": "xmldom",
-  "version": "0.4.0",
-  "lockfileVersion": 1,
+  "version": "0.4.1-dev",
+  "lockfileVersion": 2,
```

Comment:
I think the massive changes in this file are due to this change and I'm not sure if this is what we want.